### PR TITLE
Deepfried items no longer drop the unfried item when put in the reclaimer

### LINKED ISSUE #6909: Infinite fried meat/biomatter
### Labels

[BUG]

### Description

If you put fried meat into an enzymatic reclaimer it will drop a duplicated non-fried meat onto the ground. The fried meat will also be put in the reclaimer which you can eject or reclaim for biomatter. You can re-fry the unfried meat and repeat for infinite meat.

### Steps to reproduce

1. Fry meat
2. Put meat into enzymatic reclaimer
3. Fry the newly created non-fried meat
4. Put new fried meat into enzymatic reclaimer
5. repeat steps 3 and 4

### Expected Behavior

Clicking fried meat on the reclaimer does not duplicate it and somehow un-fry it.

### Additional Information & Screenshots

_No response_

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[BUG]

## About the PR 
Fixes #6909 

## Why's this needed? 
Putting a deepfried item in the enzymatic reclaimer now deletes everything inside it just like eating it would.

### CODE REVIEW

File: code/obj/machinery/clonepod.dm [hunk 1]
```diff
@@ -1043,7 +1043,7 @@
 				boutput(user, "<span class='alert'>There is already enough meat in there! You should not exceed the maximum safe meat level!</span>")
 				return
 
-			if (G.contents && G.contents.len > 0)
+			if (G.contents && G.contents.len > 0 && !istype(G, /obj/item/reagent_containers/food/snacks/shell/deepfry))
```
Comment (Zonespace27) [inline code/obj/machinery/clonepod.dm:1046]:
```suggestion
			if (G.contents && length(G.contents) > 0 && !istype(G, /obj/item/reagent_containers/food/snacks/shell/deepfry))
```

**Comment by Sovexe** [comment]
Player mobs needs to ghostize as appropriate or this will crash clients

**Comment by TobleroneSwordfish** [comment]
> E: Actually this is already being handled by the fry shell itself
> 
> ```dm
> 	disposing()
> 		for (var/mob/M in src)
> 			M.ghostize()
> 			for (var/obj/item/I in M)
> 				I.dispose()
> 		..()
> ```

Ah so that's okay then? I honestly don't know how you'd get a player mob into a deepfry shell anyway but I suppose it must be possible.

**Comment by ZeWaka** [comment]
> Ah so that's okay then? I honestly don't know how you'd get a player mob into a deepfry shell anyway but I suppose it must be possible.

suicide into fryer or some other cursed methods

File: code/obj/machinery/clonepod.dm [hunk 2]
```diff
@@ -1043,7 +1043,7 @@
 				boutput(user, "<span class='alert'>There is already enough meat in there! You should not exceed the maximum safe meat level!</span>")
 				return
 
-			if (G.contents && G.contents.len > 0)
+			if (G.contents && length(G.contents) > 0 && !istype(G, /obj/item/reagent_containers/food/snacks/shell/deepfry))
```
Comment (Tarmunora) [inline code/obj/machinery/clonepod.dm:1046]:
```suggestion
			if (G.contents && length(G.contents) > 0 && !istype(G, /obj/item/reagent_containers/food/snacks/shell))
```
grills exist too

**Pull request merged**